apps/assistants/views.py

```diff
@@ -124,6 +126,23 @@ class EditOpenAiAssistant(BaseOpenAiAssistantView, UpdateView):
     button_text = "Update"
     permission_required = "assistants.change_openaiassistant"
 
+    def get(self, request, *args, **kwargs):
+        self.object = self.get_object()
+        try:
+            is_synced = is_synced_with_openai(self.object)
```

Comment:
What's your thoughts on doing this check asynchronously? Using htmx, this should be easy enough I'd imagine. In my mind we have a sync status indicator. We can play with a few icons to indicate "checking status", "out of sync", "in sync" and "check failed". The out of sync status can be a nice scary red that draws attention. Up to you

Reply:
I like that and made the fix